templates/all-services/all-services-en.html

```diff
@@ -0,0 +1,738 @@
+			<div class="col-md-3">
+				<h3 class="mt-0"><span class="fas fa-exclamation-circle text-info mr-2" aria-hidden="true"></span>Information on</h3>
+			</div>
+			<div class="col-md-9">
+				<ul class="colcount-md-2 small bold-content lst-spcd fnt-hdng">
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/refugees/afghanistan.html">Afghanistan</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/haiti-2024.html">Haiti</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/iran.html">Iran</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/israel-west-bank-gaza-2023.html">Israel, the West Bank and the Gaza Strip</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/lebanon-2023.html">Lebanon</a>&nbsp;&nbsp;<span class="label label-primary">Updated</span></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/sudan2023.html">Sudan</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/immigrate-canada/ukraine-measures.html">Ukraine</a>&nbsp;&nbsp;<span class="label label-primary">Updated</span></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/emergencies-canada/wildfires-2024.html">Wildfires in Canada</a></li>
+				</ul>
+			</div>
+		</section>
+		<section class="bg-light row pt-3 pb-2">
+			<div class="col-md-3">
+				<h3 class="mt-0">Most requested</h3>
+			</div>
+			<div class="col-md-9">
+				<ul class="colcount-md-2 small bold-content lst-spcd fnt-hdng">
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/application/ircc-accounts.html">Sign in to your IRCC account</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/application/check-status.html">How to check your application status</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/application/check-processing-times.html">Check application processing times</a></li>
+					<li><a href="https://www.canada.ca/en/immigration-refugees-citizenship/services/application/application-forms-guides.html">Find an application form</a></li>
+					<li><a href="https://ircc.canada.ca/english/information/fees/index.asp">Pay your fees</a></li>
+					<li><a href="https://ircc.canada.ca/english/visit/visas.asp">Find out if you need an eTA or a visa to visit Canada</a></li>
+					<li><a href="https://ircc.canada.ca/english/helpcentre/index-featured-can.asp">Have questions? Find answers in the Help Centre</a></li>
+				</ul>
+			</div>
+		</section>
+		<section class="row pt-3 pb-2">
+			<div class="col-md-3">
+				<h3 class="mt-0">More topics</h3>
+			</div>
+			<div class="col-md-9">
+				<ul class="colcount-md-2 small bold-content lst-spcd fnt-hdng">
```

Comment:
May be the style `fnt-hdng` should be applied to the top container instead? Something to look into it, not required.